src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -72,6 +77,11 @@ public abstract class BatchExportProcessor<T> : BaseExportProcessor<T>
             this.scheduledDelayMilliseconds = scheduledDelayMilliseconds;
             this.exporterTimeoutMilliseconds = exporterTimeoutMilliseconds;
             this.MaxExportBatchSize = maxExportBatchSize;
+
+            this.exporterName = exporter.GetType().Name;
+            this.exportedDataType = typeof(T).Name;
+            this.processorId = Interlocked.Increment(ref processorCount);
```

Comment:
What does this id mean? (e.g. how would the user use it?)

Reply:
This is to be added as a dimension on metric (In a scenario where they have same exporterName and exportedDataType).